epol-update-repo/oe-extra-repos.repo

```diff
@@ -0,0 +1,7 @@
+[EPOL_update]
```

Comment:
`epol-update-repo/oe-extra-repos.repo` - should this also include `EPOL` URL (otherwise only EPOL update it's a little bit weird)?

Reply:
```
[EPOL]
name=EPOL
baseurl=http://repo.openeuler.org/openEuler-22.03-LTS/EPOL/main/$basearch/
enabled=1
gpgcheck=1
gpgkey=http://repo.openeuler.org/openEuler-22.03-LTS/OS/$basearch/RPM-GPG-KEY-openEuler
```

is already included in /etc/yum.repos.d/openEuler.repo. 
At least I see it in docker.io/openeuler/openeuler:22.03-lts

Reply:
Yes, I know, but....OK, I can accept. : )

Reply:
Otherwise `dnf` will report a warning that it is defined twice.